lib/modules/manager/crossplane/readme.md

```diff
@@ -0,0 +1,40 @@
+The `crossplane` manager has no `fileMatch` default patterns, so it won't match any files until you configure it with a pattern.
+This is because there is no commonly accepted file/directory naming convention for crossplane YAML files and we don't want to check every single `*.yaml` file in repositories just in case any of them have Crossplane Packages definitions: Configurations, Providers, Functions.
```

Comment:
```suggestion

There is no commonly accepted file/directory naming convention for crossplane YAML files.
We don't want Renovate to check each `*.yaml` file in repositories just in case a files has one of these Crossplane Package definitions:

- Configurations
- Providers
- Functions
```